### DISCUSSION TITLE
[Suggestions] User controlled experience

### DISCUSSION THREAD

**Discussion by dmadisetti** [SebLague/Geographical-Adventures#53] [Ideas]
Sebastian. I have so many thoughts, this is really fun. I think I might fork this project and start tinkering with it myself, but here are outstanding ideas that I thought were worth sharing:

 - Option to decrease curvature of the map. The world is big, but I think it could afford to be a bit bigger, especially when trying to target island nations or getting <50km accuracy on cities. For reference, the ISS circles at 408 km
 - Remove boundaries. I do like the light up effect, I think there's benefit in keeping that- but the world doesn't have boundaries (maybe as a 'hard mode option'). The current boundaries are not super useful for instance for Europe at night.

Game play mode ideas
- Timed mode
- All countries mode 

Map "games" I enjoy:
- Sporcle: https://www.sporcle.com/games/teedslaststand/find-world-countries)
- On android [Map World Quiz](http://worldmapquiz.qbisstudio.com/)

FYI, you can also start Github Discussions: https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions
If you'd prefer more a chat medium over issues.

**Comment by SebLague** [comment]
Thanks for the suggestions, and I've enabled Discussions now :)
